client/app/views/home_application.coffee

```diff
@@ -127,12 +127,12 @@ module.exports = class ApplicationRow extends BaseView
             when 'installing'
                 alert t 'state app installing'
             when 'stopped'
-                @startApp()
+                @startApp(event)
 
 
     ### Functions ###
 
-    startApp: =>
+    startApp: (e) =>
```

Comment:
@clochix are you sure it should be called `e` and not `event`? It doesn't look like `e` is used in this function, but there is an `event`